packages/nextjs/components/Header.tsx

```diff
@@ -18,6 +18,12 @@ export const menuLinks: HeaderMenuLink[] = [
   {
     label: "Home",
     href: "/",
+    icon: <HomeIcon className="h-4 w-4" />,
+  },
+  {
+    label: "Builders",
+    href: "/builders",
+    icon: <UsersIcon className="h-4 w-4" />,
```

Comment:
Same, let's leave this for https://github.com/BuidlGuidl/batch7.buidlguidl.com/issues/3